libres/lib/include/ert/enkf/hook_manager.hpp

```diff
@@ -55,4 +56,24 @@ hook_manager_iget_hook_workflow(const hook_manager_type *hook_manager,
                                 int index);
 extern "C" int hook_manager_get_size(const hook_manager_type *hook_manager);
 
+/** An element in the runpath list file */
+class Runpath {
```

Comment:
The functions in this class are all very trivial. Inline their definitions into the header.

Reply:
Done